exampleHist.py

```diff
@@ -51,9 +52,11 @@ def push(self, event):
         orders.append(Order(event.instrument, -1, 0))
         self.signal = -1
 
+
       del self.prices[0]
 
     return orders
 
-print(evaluateHist(BuynHold(), {'IBOV':'^BVSP.csv'}))
-print(evaluateHist(MAVG(), {'IBOV':'^BVSP.csv'}))
\ No newline at end of file
+#print(evaluateHist(BuynHold(), {'IBOV':'^BVSP.csv'}))
+#print(evaluateHist(MAVG(), {'IBOV':'^BVSP.csv'}))
```

Comment:
Comment code should not be pushed into production. Also, why are you changing the exampleHist.py file? Your changes seem disrupt it's expected functionality.

Reply:
> Comment code should not be pushed into production. Also, why are you changing the exampleHist.py file? Your changes seem disrupt it's expected functionality.

A lot of things wrong here. First of all: Changes do not disrupt the main code, but I agree that exampleHist.py should not be changed. Second: Comments are not allowed in production? That doesn't make any sense since the README doesn't explain exactly how tests can be done. So, I commented the tests in my file, but feel free to alter the README and explain how tests work, then I will agree with you.  Last of all, I think you are not sure of all things you are saying, although you are trying to help. Please, read the code and test it just like everybody does before making such accusations. Thanks

![tungada](https://user-images.githubusercontent.com/11521474/58249624-c0a09980-7d35-11e9-81c8-9b7e9ba6e9f1.jpg)